fix: prevent failing in worker (#1478)
Redocly/redocly-cli: .changeset/moody-parents-fry.md

---
"@redocly/openapi-core": patch
---

> Added a platform check so `@redocly/openapi-core` doesn't fail when running inside a worker.

Reviewer: ```suggestion
Added a platform check so `@redocly/openapi-core` can support running inside a worker.
```
